### PR TITLE
RDKB-Resolving Components Dependency of rdkcentral RBUS

### PR DESCRIPTION
**Pull request by NetajiPanigrahi**
RDKB-45117: RDKB-Resolving Components Dependency of rdkcentral RBUS
Reason for change: Resolve build issue
Test Procedure: making changes on TECHNICOLORXB7 and checking "comcast-broadband-dev-image" and resolving component dependencies with Rbus

Signed-off-by: Netaji Panigrahi <Netaji_Panigrahi@comcast.com>

### CODE REVIEW

**Review by karuna2git — APPROVED** [review]

File: src/webcfg_rbus.h [hunk 1]
```diff
@@ -26,8 +26,7 @@
 #include <rbus/rbus_object.h>
 #include <rbus/rbus_property.h>
 #include <rbus/rbus_value.h>
-#include <rbus-core/rbus_core.h>
-#include <rbus-core/rbus_session_mgr.h>
```
Comment (shilpa24balaji) [inline src/webcfg_rbus.h:30]: Is this session_mgr header not available now?

File: CMakeLists.txt [hunk 2]
```diff
@@ -54,9 +54,9 @@ endif (FEATURE_SUPPORT_AKER)
 
 if (WEBCONFIG_BIN_SUPPORT)
 include_directories(${INCLUDE_DIR}/rbus
-		    ${INCLUDE_DIR}/rbus-core
 		    ${INCLUDE_DIR}/rtmessage
```
Comment (shilpa24balaji) [inline CMakeLists.txt:57]: Is rtmessage include delete not needed as in here https://github.com/xmidt-org/parodus/pull/407/files#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL49

File: src/webcfg_rbus.h (hunk 1 above)
Comment (shilpa24balaji) [inline src/webcfg_rbus.h:30]: Is this session_mgr header not available now?

Comment (NetajiPanigrahi) [reply src/webcfg_rbus.h:30]: Available, " #include rbus/rbus_session_mgr.h"
Use session manager wapper APIs in rbus.h header file.

File: CMakeLists.txt [hunk 3]
```diff
@@ -187,44 +187,22 @@ add_dependencies(liblibparodus libparodus)
...
-     CMAKE_ARGS += -DBUILD_FOR_DESKTOP=ON -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR}
-                   -DBUILD_TESTING=OFF
-)
-add_library(librbus-core STATIC SHARED IMPORTED)
-add_dependencies(librbus-core rbus-core)
-
 # rbus external dependency
 #-------------------------------------------------------------------------------
 ExternalProject_Add(rbus
      DEPENDS rtMessage rbus-core
```
[the remark this answers is not in the document]
Comment (NetajiPanigrahi) [reply CMakeLists.txt:193]: I will delete these depends

[the remark this answers is not in the document]
Comment (shilpa24balaji) [reply CMakeLists.txt:193]: ok please update this PR with this change

**Pull request closed without merging**